cmd/kaspawallet/libkaspawallet/bip39.go

```diff
@@ -16,7 +16,8 @@ func CreateMnemonic() (string, error) {
 	return bip39.NewMnemonic(entropy)
 }
 
-func defaultPath(isMultisig bool) string {
+// DefaultPath returns the
```

Comment:
What does it return?

Reply:
Anyway, I'd prefer if you won't export it. If you need it to determine what is kaspa coin type you can just export the coin type